### PR TITLE
chore(grants): estimate clr via celery

### LINKED ISSUE #7854: move estimate clr to celery task
#### Description 

speed up grant clr calcs - if we want to speed up the calcs, i could see offloading the `estimate_clr`jobs to the celery queues tho.   that’d be an elegant way to paralellize them

- update mgmt command estimate.clr to invoke predict_clr as a celery task 
- when you haven multiple CLR rounds -> all-round calculations should be offloaded to celery task

note:
In prod -> we can create more celery workers to make the task run in parallel

### PR DESCRIPTION
**Pull request by chibie**
##### Description

This moves invocation of `predict_clr` by `estimate clr` management command to a celery task

##### Refers/Fixes

closes #7854

##### Testing

image

### CODE REVIEW

**Comment by chibie** [comment]
@thelostone-mc all set here?

File: app/grants/tasks.py [hunk 1]
```diff
@@ -206,6 +212,19 @@ def recalc_clr(self, grant_id, retry: bool = True) -> None:
...
+@app.shared_task(bind=True, max_retries=1)
+def process_predict_clr(save_to_db=False, from_date=None, clr_round=None, network='mainnet', only_grant_pk=None):
+    from grants.clr import predict_clr
+    return predict_clr(
+        save_to_db=False,
+        from_date=None,
+        clr_round=None,
+        network='mainnet',
+        only_grant_pk=None
+    )
```
Comment (thelostone-mc) [inline app/grants/tasks.py:225]: ^ I don't get why we are doing this?

Comment (thelostone-mc) [reply app/grants/tasks.py:225]: Did you test this out @chibie ? 
Could we pass only the needed arguments as is

Comment (thelostone-mc) [reply app/grants/tasks.py:225]: Also do we need a return ?

File: app/grants/management/commands/estimate_clr.py [hunk 2]
```diff
@@ -49,14 +49,14 @@ def handle(self, *args, **options):
...
                     save_to_db=True,
                     from_date=timezone.now(),
                     clr_round=clr_round,
                     network=network
                 )
-                print(f"finished CLR estimates for {clr_round.round_num}")
+                print(f"finished CLR estimates for {clr_round.round_num} {clr_round.sub_round_slug}")
 
                 # TOTAL GRANT
                 # grants = Grant.objects.filter(network=network, hidden=False, active=True, link_to_new_grant=None)
```
Comment (thelostone-mc) [inline app/grants/management/commands/estimate_clr.py:62]: ^ I believe we'd have to move this commented out code as well into process_predict_clr
this is used for debugging

**Comment by thelostone-mc** [comment]
@chibie could you through in a screenshot as well just to make sure this works as expected and the logs are printed in order and stuff

File: app/grants/tasks.py (hunk 1 above)
Comment (chibie) [reply app/grants/tasks.py:225]: `predict_clr` function in clr.py returns `debug_output`

File: app/grants/management/commands/estimate_clr.py (hunk 2 above)
Comment (chibie) [reply app/grants/management/commands/estimate_clr.py:62]: oh right!

File: app/grants/tasks.py (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/tasks.py:225]: Ah i don't know what we use that for ! 
Cause if you look at other places -> we don't do anything with it

Comment (thelostone-mc) [reply app/grants/tasks.py:226]: could we switch it to 
```
process_predict_clr(save_to_db, from_date, clr_round, network):

    predict_clr(
        save_to_db,
        from_date,
        clr_round,
        network
    )
```

File: app/grants/management/commands/estimate_clr.py [hunk 3]
```diff
@@ -49,24 +49,13 @@ def handle(self, *args, **options):
...
-                # grants = Grant.objects.filter(network=network, hidden=False, active=True, link_to_new_grant=None)
-                # grants = grants.filter(**clr_round.grant_filters)
-
-                # total_clr_distributed = 0
-                # for grant in grants:
-                #     total_clr_distributed += grant.clr_prediction_curve[0][1]
-
-                # print(f'Total CLR allocated for {clr_round.round_num} - {total_clr_distributed}')
-
+                print(f"finished CLR estimates for {clr_round.round_num} {clr_round.sub_round_slug}")
```
Comment (thelostone-mc) [inline app/grants/management/commands/estimate_clr.py:59]: This print statement also would have to be pushed into it right ?

**Pull request merged**